apps/i18n/common/en_us.json

```diff
@@ -368,8 +368,7 @@
   "unnamedFunction": "You have a variable or function that does not have a name. Don't forget to give everything a descriptive name.",
   "unattachedBlockTipTitle": "Unattached blocks",
   "unattachedBlockTipBody": "Blocks that are not attached will not do anything. If you want these blocks in your program, try connecting them to other blocks.",
-  "unplugged": "UNPLUGGED",
```

Comment:
Yeah, the usage was removed in an earlier PR and I had neglected to remove the string.